docs/usage/self-hosted-configuration.md

```diff
@@ -7,6 +7,8 @@ description: Self-Hosted Configuration usable in renovate.json or package.json
 
 The below configuration options are applicable only if you are running your own instance ("bot") of Renovate.
 
+Please also see [Self-Hosted Experimental Options](./self-hosted-experimental.md).
```

Comment:
Should we add this to the sidebar too ?

Reply:
I thought best not to. We already have a lot, and the whole idea of them is to keep them a little out of sight :)